README.md

```diff
@@ -58,9 +58,9 @@ The following environment variables are read by **define**'s sources:
 
 ## Sources
 
-The **define** app has access to multiple sources, however some of them require user-specific API keys, due to usage limitations.
+The **define** app has access to multiple sources, however some require user-specific API keys due to usage limitations.
 
-You can specify a preferred source either via the command line flag `--preferred-source="..."` or in your configuration file. For more information, see the section on [Configuration](#configuration).
+A preferred source can be specified with the command line flag `--preferred-source="..."` or in a configuration file. For more information, see the section on [Configuration](#configuration).
```

Comment:
I like the change away from "You can specify" and "your", as its less awkwardly "personal", but otherwise I'm not a fan of these changes. I'd be willing to merge this if you just cut down the changes to removing the personal bits but not changing the other language. 😃